README.adoc

```diff
@@ -22,11 +22,11 @@ Substrate is still an early stage project, and while it has already been used as
 
 Substrate is designed to be used in one of three ways:
```

Comment:
Without this change, you get an English _type error_ in the list, i.e. "generic" should be "generically," etc.
```suggestion
Substrate is designed for use in one of three ways:
```